packages/boxel/addon/components/boxel/modal/usage.hbs

```diff
@@ -25,7 +26,8 @@
 </Boxel::Modal>
 </:example>
 <:api as |Args|>
-  <Args.String @name="size" @description="Can be `small`, `medium`, `large`, or unspecified. Sets a predetermined max-width to inner modal." @value={{this.size}} @options={{array "small" "medium" "large" "n/a"}} @onInput={{fn (mut this.size)}} @defaultValue="65rem" />
+  <Args.String @name="size" @description="Can be `small`, `medium`, `large`, or unspecified. Sets a predetermined max-width to inner modal." @value={{this.sizeAsString}} @options={{array "small" "medium" "large" "<undefined>"}} @onInput={{this.updateSize}} @defaultValue="65rem" />
```

Comment:
Is this `defaultValue` appropriate here? `65rem`  might also be confusing when the options are strings, not CSS lengths.

Reply:
I actually think maybe this isn’t meant to be overridden…? I got that from [here](https://github.com/cardstack/cardstack/blob/05b0f342a863670d0a8b019d1770eb0d21ea9e6a/packages/boxel/addon/components/boxel/modal/index.css#L50) but it’s true that it’s superseded if a `size` is set.

Reply:
oh I misunderstood. No, I don’t think this should be `65rem`.